cmake/BundledOpenCV.cmake

```diff
@@ -138,10 +134,15 @@ function(use_bundled_opencv SOURCE_DIR BINARY_DIR)
     set_target_properties(OPENCV::libpng PROPERTIES IMPORTED_LOCATION "${OPENCV_BYPRODUCT_DIR}/${CMAKE_INSTALL_LIBDIR}/${THIRDPARTY_DIR}${PREFIX}libpng${THIRDPARTY_SUFFIX}")
     add_dependencies(OPENCV::libpng opencv-external)
 
+    add_library(OPENCV::libopenjp2 STATIC IMPORTED)
```

Comment:
are there some PRs incoming that verify this fix?

Reply:
OpenCV is compiled in the CI so the upgrade should be okay but I only tested the loading of the library manually. The proper way would be to have some test coverage for the OpenCV extension, but currently I'm not aware of anyone using it so it is not a priority. I created a ticket for this in Jira so we can add it in the future https://issues.apache.org/jira/browse/MINIFICPP-1709